src/Tooltip/Tooltip.style.js

```diff
@@ -0,0 +1,21 @@
+export default {
+  backgroundColor: 'lighterText',
+  borderRadius: '8px',
+  padding: '8px 11px',
```

Comment:
I'm talking to Lorina to add spacing to the styleguide. So essentialy we'll have an array 
```jsx
theme.space = ["2px", "4px", "8px", "12px", "16px", "24px", "32px", "40px", "48px"...]
```
And where there's space to be used we'll use an index from here so: `padding: 2`. Designers will also always use margins and paddings from this array.

No need to change now but once we add that space array we'll have to comb and replace everywhere.